src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -824,6 +824,21 @@ def load_arguments(self, _):
                    help="swap types. use 'preview' to apply target slot's settings on the source slot first; use 'swap' to complete it; use 'reset' to reset the swap",
                    arg_type=get_enum_type(['swap', 'preview', 'reset']))
 
+    with self.argument_context('functionapp deployment github-actions')as c:
+        c.argument('name', arg_type=functionapp_name_arg_type)
+        c.argument('resource_group', arg_type=resource_group_name_type, options_list=['--resource-group', '-g'])
```

Comment:
`options_list=['--resource-group', '-g']` is already defined in `resource_group_name_type`, so it does not need to be defined repeatedly
```suggestion
        c.argument('resource_group', arg_type=resource_group_name_type)
```